test/BUILD.gn

```diff
@@ -573,10 +573,6 @@ test("brave_unit_tests") {
     ]
   }
 
-  if (enable_speedreader) {
-    deps += [ "//brave/components/speedreader:unit_tests" ]
```

Comment:
@bridiver do we run components unit tests on Android now? I only see `brave_unit_tests` on CI.

Reply:
now we do. I see speedreader tests in the test report on CI.